src/battle_transition.c

```diff
@@ -3912,17 +3915,28 @@ void Task_BattleTransition_Intro(u8 taskId)
 
 static bool8 TransitionIntro_FadeToGray(struct Task *task)
 {
+    u8 paletteNum = IndexOfSpritePaletteTag(TAG_WEATHER_START);
+    u16 index = OBJ_PLTT_ID(paletteNum) + 9; // SHADOW_COLOR_INDEX
```

Comment:
Why isn't the define used here instead of the magic number?

Reply:
Fixed